crawl-ref/source/dat/des/branches/coc.des

```diff
@@ -193,6 +193,548 @@ xxxxxxxxxxxxxxxxxxxxxxxxx.<xxxxxxxxxxxxxxx
+xxM..{..Mxx
+xxxx~~~xxxx
+ xxxxxxxxx
+ENDMAP
+
+NAME:  nicolae_coc_melting_block
+TAGS:  transparent allow_dup
+DEPTH: Coc
+SUBST: M = mxx, N = Wmm, O = .WW
+MAP
+..........
+.OOOOOOOO.
+.ONNNNNNO.
+.ONMMMMNO.
+.ONMxxMNO.
+.ONMxxMNO.
+.ONMMMMNO.
+.ONNNNNNO.
+.OOOOOOOO.
+..........
+ENDMAP
+
+NAME:    nicolae_coc_naught_crossing
+TAGS:    transparent no_pool_fixup
+DEPTH:   Coc
+ORIENT:  centre
+SHUFFLE: w.
+MAP
+..xxxwwwwwxxx..
+...wwwwwwwww...
+x.WWwwwwwwwWW.x
+xwWWWxxxxxWWWwx
+xwwW..xxx..Wwwx
+wwwx...x...xwww
+wwwxx.....xxwww
+wwwxxx...xxxwww
+wwwxx.....xxwww
+wwwx...x...xwww
+xwwW..xxx..Wwwx
+xwWWWxxxxxWWWwx
+x.WWwwwwwwwWW.x
+...wwwwwwwww...
+..xxxwwwwwxxx..
+ENDMAP
+
+NAME:    nicolae_coc_on_the_berg
+TAGS:    transparent no_pool_fixup
+DEPTH:   Coc, !Coc:$
+ORIENT:  encompass
+KFEAT:   E = open_sea
+: set_border_fill_type('open_sea')
+SHUFFLE: Qq, Or
+NSUBST:  S = { / } / !
+SUBST:   ~ = w., D : w-, F : w-, H : w-, J : w-, K : w-, L : w-, \
+         M : xxw--, N : xxw--, O : xxw--, P : xxw--, Q : xxw--, \
+         R : xxw--, p : xxw--, qr : -, - = .:40 !:1 8:1
+MARKER:  ! = lua:fog_machine { pow_max = 20, cloud_type = "freezing vapour", \
+             delay_min = 50, delay_max = 250, size = 16, start_clouds = 1, \
+             excl_rad = -1 }
+MAP
+EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE
+EwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwE
+EwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwE
+Ewwwww~...DDDwwwwwwww~....~wwwwwwwE
+Ewww~......DDDDwwww~...MM.....~wwwE
+Ewww..S.....DDDDDDD...MMMM.....~wwE
+Eww~.........DDDDD...MMMMM...S..wwE
+Eww..........DDDDDD..MMMMM......wwE
+Eww..pppp...DDDDDDD...MMM......~wwE
+EwwLppppp....DDDDD...RRR......~wwwE
+EwwLLppppp....DDD...RRRRR.....wwwwE
+EwwLLLppp...........RRRRR...FFFwwwE
+EwwwLLLLL.........qqRRRRRFFFFFFFwwE
+EwwwwLLLLL....xxxqqqqRRRFFFFFFFFwwE
+EwwwLLLLLL...xxxxxxqq...FFFFFFFFwwE
+EwwLLLL....QQxxxxxxq....FFFFFFFFwwE
+EwwL.....QQQQQxxxx......NNFFFFFFwwE
+Eww......QQQQQQ.........NNNNNFFwwwE
+EwwK..KKKKKQQxx.!.rrxxx..NNNNNFwwwE
+EwwKKKKKKKKKxxxxrrrxxxxx..NNNHwwwwE
+EwwwKKKKKKKKxxxxrrxxxxx...HHHHwwwwE
+EwwwwKKKKKKxxxxxrrxxxxx..HHHHHHwwwE
+EwwwwKKKKK..xxx..rrrxxx..HHHHHHHwwE
+EwwwwKK..............xxxOOHHHHHHwwE
+EwwwK....PP...JJJJJJ..xxOOO.HHHHwwE
+Ewww....PPPP.JJJJJJJJ.OOOOO...HwwwE
+Eww~...PPPPPJJJJJJJJJ..OOO.....~wwE
+Eww.....PPPPJJJJJJJJ...OO.......wwE
+Eww......PP..JJJJJ..........S..~wwE
+Eww..S.........JJ.............~wwwE
+Eww~....~ww~...JJJ..~www~....~wwwwE
+Ewww~..~wwww~.JJJJJwwwwwwwwwwwwwwwE
+EwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwE
+EwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwE
+EEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE
+ENDMAP
+
+NAME:    nicolae_coc_statues_of_limitations
+TAGS:    transparent allow_dup luniq
+DEPTH:   Coc, !Coc:$
+KMONS:   1 = ice statue
+KFEAT:   1 = W
+KFEAT:   ^ = shaft trap
+SHUFFLE: DE
+SUBST:   D : 1, E : GG1, } = }}^
+MAP
+WWWWW
+WDEDW
+WE}EW
+WDEDW
+WWWWW
+ENDMAP
+
+NAME:   nicolae_coc_three_more_mouths
+TAGS:   transparent patrolling no_pool_fixup
+DEPTH:  Coc, !Coc:$
+ORIENT: centre
+KMONS:  1 = ice fiend
```

Comment:
TBH, I tend to avoid using MONS: statements, since if I want to change the glyph for a monster I have to rearrange all the glyphs for the other monsters that were numbered after it. Also it's easier for me to skim a vault def, since the glyph is right next to the monster definition. (I can still change it if it's necessary for Vault Consistency, though.)

Reply:
Ah, yeah ok. That's reasonable.